test/drb/test_drb.rb

```diff
@@ -333,6 +340,41 @@ def test_05_too_large_ary
     end
     assert_kind_of(StandardError, exception)
   end
+
+  def test_06_array_operations
+    ary = [1,50,3,844,7,45,23]
+    assert_equal(7, @there.size(ary))
+    assert_equal(973, @there.sum(ary))
+    assert_equal(917217000, @there.multiply(ary))
+    assert_equal(139.0, @there.avg(ary))
+    assert_equal(23.0, @there.median(ary))
+
+    ary2 = [1,2,3,4]
+    assert_equal(4, @there.size(ary2))
+    assert_equal(10, @there.sum(ary2))
+    assert_equal(24, @there.multiply(ary2))
+    assert_equal(2.5, @there.avg(ary2))
+    assert_equal(2.5, @there.median(ary2))    
+
+  end
+
+  def test_07_one_element_array
```

Comment:
@niroyb - thanks for the edits - the only thing I didn't change was renaming <code>multiply</code> to <code>product</code> 

- originally I thought <code>product</code> would be better too, but then I thought the name might get confused with ruby's existing cartesian product method - http://ruby-doc.org/core-2.2.0/Array.html#method-i-product